proxy/src/control/mod.rs

```diff
@@ -22,6 +22,7 @@ use timeout::{Timeout, TimeoutError};
 
 mod cache;
 pub mod discovery;
+mod fully_qualified_authority;
```

Comment:
I moved fully_qualified_authority to be closer to `control::discovery` since `control::discovery` is the only user of it now. I didn't make it a sub-module of `control::discovery` because I didn't want to rename `discovery.rs` to `discovery/mod.rs`.